44/README.md

```diff
@@ -0,0 +1,140 @@
+1. Thus, the release history of the project SHALL be a list of meaningful issues logged and solved.
+1. To work on an issue, a Contributor SHALL fork the project repository and then work on their forked repository.
+1. To submit a patch, a Contributor SHALL create a Platform pull request back to the project.
+1. A Contributor SHALL NOT commit changes directly to the project.
+1. If the Platform implements pull requests as issues, a Contributor MAY directly send a pull request without logging a separate issue.
+1. To discuss a patch, people MAY comment on the Platform pull request, on the commit, or elsewhere.
+1. To accept or reject a patch, a Maintainer SHALL use the Platform interface.
+1. Maintainers SHOULD NOT merge their own patches except in exceptional cases, such as non-responsiveness from other Maintainers for an extended period (more than 1-2 days).
+1. Maintainers SHALL NOT make value judgments on correct patches.
+1. Maintainers SHALL merge correct patches from other Contributors rapidly.
+1. Maintainers MAY merge incorrect patches from other Contributors with the goals of (a) ending fruitless discussions, (b) capturing toxic patches in the historical record, (c) engaging with the Contributor on improving their patch quality.
+1. The user who created an issue SHOULD close the issue after checking the patch is successful.
+1. Any Contributor who has value judgments on a patch SHOULD express these via their own patches.
+1. Maintainers SHOULD close user issues that are left open without action for an uncomfortable period of time.
+
+### 2.5. Branches and Releases
+
+1. The project SHALL have one branch ("master") that always holds the latest in-progress version and SHOULD always build.
+1. The project SHALL NOT use topic branches for any reason. Personal forks MAY use topic branches.
+1. To make a stable release a Maintainer shall tag the repository. Stable releases SHALL always be released from the repository master.
+
+### 2.6. Evolution of Public Contracts
+
+1. All Public Contracts (APIs or protocols) SHALL be documented.
+1. All Public Contracts SHOULD have space for extensibility and experimentation.
+1. A patch that modifies a stable Public Contract SHOULD not break existing applications unless there is overriding consensus on the value of doing this.
+1. A patch that introduces new features SHOULD do so using new names (a new contract).
+1. New contracts SHOULD be marked as "draft" until they are stable and used by real users.
+1. Old contracts SHOULD be deprecated in a systematic fashion by marking them as "deprecated" and replacing them with new contracts as needed.
+1. When sufficient time has passed, old deprecated contracts SHOULD be removed.
+1. Old names SHALL NOT be reused by new contracts.
+1. A new contract marked as "draft" MUST not be changed to "stable" until all the following conditions are met:
+    1. Documentation has been written and finalised.
```

Comment:
What do you mean by "finalised"? Can this be formalized at all? I would drop "and finalised".

Maybe require that documentation is as comprehensive as that of comparable contracts? E.g., when there is a manpage for all existing API functions, a new API function should also have a manpage.

Reply:
Yes that's what I was meaning to convey, fixed